test/test_pipeline/test_tabular_classification.py

```diff
@@ -279,3 +282,29 @@ def test_set_range_search_space_updates(self, fit_dictionary):
             # As we are setting num_layers to 1 for fully connected
             # head, units_layer does not exist in the configspace
             assert 'fully_connected:units_layer' in e.args[0]
+
+    def test_trainer_cocktails(self, fit_dictionary, mocker):  # noqa F811
+        fit_dictionary['epochs'] = 10
+        pipeline = TabularClassificationPipeline(
+            dataset_properties=fit_dictionary['dataset_properties'],
+            include={'lr_scheduler': ['CosineAnnealingWarmRestarts']})
+        cs = pipeline.get_hyperparameter_search_space()
+        config = cs.get_default_configuration()
```

Comment:
Can you add a check to make sure CosineAnnealingWarmRestarts is in the config below?

Reply:
sure